### DISCUSSION TITLE
"Paid" segment in Packages page

### DISCUSSION THREAD

**Discussion by kelvin1306** [wstyres/Zebra#276] [Ideas]
hey I have a new idea to make Zebra 🦓 look more modern. Look this picture 👇 in the top where "ABC" and "Date" make a new one that says *Paid Tweaks* that will look awesome 👏 and not other package manager will has this feature. 👌🤷‍♂️ 
![05CD49C9-DCBB-4E6A-B375-B7D7653E5AF0](https://user-images.githubusercontent.com/52043094/61820748-ab261980-ae23-11e9-9e28-54c3c1a2a23b.jpeg)

Also make this for *Repository* for example: 
iospackixrepo and others like this 👇👀 that have Paid Tweaks. Thanks for everything, this is the best package manager with this feature. 👏🤷‍♂️🔥
![549A2724-0C05-4D19-A631-4DCAC845C597](https://user-images.githubusercontent.com/52043094/61820865-e6c0e380-ae23-11e9-9c92-046a244c55e2.jpeg)

Can you also fix this problem I try to download Magma from @NoisyFlake but this want remove other tweaks. I had 👉libcolorpicker* from @bigboss repo, create by @PixelFireDev. Thanks @xTM3x, @PoomSmart and the others dev behind this awesome 👏 package. 📦🙏
![622DC5F1-6071-4B25-9A33-068FD674D11E](https://user-images.githubusercontent.com/52043094/61820912-03f5b200-ae24-11e9-95e0-b814524c1521.jpeg)
![4986150F-5E09-4603-9738-806AC9DABA4E](https://user-images.githubusercontent.com/52043094/61820913-03f5b200-ae24-11e9-9595-681883eb0d56.jpeg)

Look I try to download from Cydia on ios 12.1.1 iPhone 8 plus it working correctly. Take a look 👀🙏👇
![E236C913-D9CC-49FF-9020-41ED7FF3C73D](https://user-images.githubusercontent.com/52043094/61820958-1f60bd00-ae24-11e9-8e55-c591f241bf4e.jpeg)
![95F0611C-DEBE-4DEC-9E02-C1BAB91690C6](https://user-images.githubusercontent.com/52043094/61820959-1ff95380-ae24-11e9-9b8c-c9663d07aa04.jpeg)

Can you also fix this bugs when I try to go down in zebra to see the description of the tweaks,, Zebra stop completely then I has to go outside to refresh the tweak. 👇🦓👏🔥🚀🤷‍♂️👀

Thanks 🙏 I just be waiting for the next update.

**Comment by PoomSmart** [comment]
The libcolorpicker Queue bug has been fixed in the next beta. The last bug you mentioned, I could not reproduce - maybe your device has other conflicting tweaks? The rest are enhancement so I will change the tag of this issue.
